### PR TITLE
Sign-up page

### PR DESCRIPTION
**Pull request by jnajera456**
### Trello
[Ticket](https://trello.com/c/2ZNbjG6i)

## Description
- added the UI for the sign up page
- added a login page anchor for existing users
- used semantic-ui
- used LabeledInput component
- made UI layout for sign up page adapt to screen size

## Screenshots
- First screenshot: desktop screen size
- Second screenshot: iphone 12 pro screen size

## TO-DO
- integrate with API - [ticket](https://trello.com/c/05TQruWy)

## Screenshots
Screen Shot 2023-08-03 at 1 00 18 PM

Screen Shot 2023-08-03 at 1 00 34 PM

### CODE REVIEW

File: pages/signUp.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import Head from 'next/head';
```
Comment (angielamm) [inline pages/signUp.js:1]: I noticed that the login.js and login.css files are included in this PR. However, I see that they are not on the same branch. Just wanted to double-check if this was intentional or if there might have been a mix-up

**Review by angielamm — COMMENTED** [review]
Awesome work! I must say, you're learning fast and doing a fantastic job  🎉

**Review by pablonajera — CHANGES_REQUESTED** [review]

File: pages/signUp.js (hunk 1 above)
Comment (jnajera456) [reply pages/signUp.js:1]: Thanks for pointing that out. It was not intentional and I'll fix it.

File: styles/signUp.module.css [hunk 2]
```diff
@@ -0,0 +1,51 @@
+.segment {
+  margin: 0px 50px !important;
+  width: 680px !important;
+  padding: 50px 75px !important;
+}
+
+@media only screen and (max-width: 754px) {
+  .pageTitle {
+    justify-content: flex-start !important;
+  }
```
[the remark this answers is not in the document]
Comment (jnajera456) [reply styles/signUp.module.css:10]: left, and ok I'll make the changes

**Comment by jnajera456** [comment]
I'm having trouble pushing my changes because my branch is behind on commits. I've tried the commands for pulling changes this is what it says:
Screen Shot 2023-08-15 at 10 51 28 AM

File: pages/signUp.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+import signupStyles from '@app/styles/signUp.module.css';
+import { Header, Segment, Form, Button } from 'semantic-ui-react';
+import cx from 'classnames';
+import LabeledInput from '@app/components/pieces/labeledInput';
+
+export default function signUp() {
```
[the remark this answers is not in the document]
Comment (angielamm) [reply pages/signUp.js]: Yes ! You just stage them and commit like the rest

[the remark this answers is not in the document]
Comment (jnajera456) [reply pages/signUp.js]: Still having trouble. I tried staging them but the renamed files aren't showing up in my source control (list of changed files)

**Review by pablonajera — APPROVED** [review]

**Pull request merged**